### PR TITLE
Updates widget boilerplate

### PR DESCRIPTION
**Pull request by tonytlwu**

### CODE REVIEW

**Comment by squallstar** [comment]
@tonytlwu can you bump up the package.json version? so I can publish afterwards

**Comment by tonytlwu** [comment]
@squallstar Approve again please 👍

File: widget-template/css/build.css [hunk 1]
```diff
@@ -1,3 +0,0 @@
-button {
```
Comment (squallstar) [inline widget-template/css/build.css]: @tonytlwu you need to remove this file from the widget.json

**Comment by tonytlwu** [comment]
@squallstar Again please :) Thanks

**Pull request merged**